### ISSUE TITLE
Docker

### ISSUE THREAD

**Issue by thany** [lixmal/keepass4web#5]
Looking at the installation guide, this project could *greatly* benefit from having a Docker implementation. Such a complicated guide is just screaming for a Dockerfile. For those unfamiliar with Docker: basically it's a way to install an application, and have it work, always. Super easy to update as well. Works on any Docker-capable computer, which is practically every Linux box, even virtualized ones.

**Comment by NiJOvanOstow** [comment]
Just created one, because i needed it ;) https://hub.docker.com/r/nijo89/keepass4web/

**Comment by lixmal** [comment]
@NiJOvanOstow thanks a lot, I'll list it in the readme.
I have yet to investigate how to avoid the issue with the same keyring for multiple containers on the same host.

**Comment by NiJOvanOstow** [comment]
yes keyring is kinda pain in docker, took me a few hours till i understood why keepass4web is working in a vm but not a docker container.

**Comment by hacker-h** [comment]
> 
> 
> Just created one, because i needed it ;) https://hub.docker.com/r/nijo89/keepass4web/

Could you provide the used repository or at least the Dockerfile please?

**Comment by NiJOvanOstow** [comment]
Just uploaded everything on GitHub: https://github.com/NiJOvanOstow/keepass4web-docker
Would be nice to create a pullrequest for me if you improve/update something ^^

**Comment by hacker-h** [comment]
It would be nice to have a Dockerfile which builds the project from source and runs out of the box e.g. without an additional LDAP service.
